src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +144,53 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const entry = $(e.target);
+        const studentEmail = $('input[name=studentEmail]').val();
+        const subjectType = $(entry).find('input[name=subjectType]').val();
+        const courseName = $(entry).find('input[name=courseName]').val();
+        const courseId = $(entry).find('input[name=courseId]').val();
+        const studentName = $(entry).find('input[name=studentName]').val();
+        const sessionStatus = $(entry).find('input[name=sessionStatus]').val();
```

Comment:
I realise I should probably have mentioned the preferred solution in my comment 😅 when using a jQuery object multiple times (as is the case here for `$(entry)`), it is preferable to extract the jQuery object as a variable first (prefixing the variable name with a $ to indicate it is a jQuery object), and then reuse that variable in later calls eg:

```js
const $entry = $(e.target);
const studentEmail = $entry.parentsUntil...
const subjectType = $entry.find...
...
```

The main difference between declaring `entry = e.target`, followed by using `$(entry)` later, and declaring `$entry = $(e.target)` and reusing `$entry` would be efficiency (no need to create the same object multiple times in the latter case).